config.py.template

```diff
@@ -473,3 +473,7 @@ class Config(object):
 
     # Excluded location for calculating geo location
     EXCLUDED_LOCATION = ast.literal_eval(os.getenv("EXCLUDED_LOCATION", "{'zoom', 'online'}"))
+
```

Comment:
`json` package needs to be added to imports here https://github.com/rokwire/events-manager/pull/572/files#diff-fcd0d12fe2d13e3d882b2e305a4069ccbee0868dc794113d7aaf6b829b747417R16